grafana/grafana
# Chore: legacy access control cleanup for frontend team pages

**What is this feature?**

Clean up legacy access control related code from the frontend team pages. Main changes:
* removed `TeamMembers.tsx` and `TeamMembersRow.tsx` - these are now handled by `TeamPermissions.tsx`;
* removed legacy access control fallbacks from team related code;
* fixed the access control logic for showing team role picker.

**Why do we need this feature?**

Legacy access control can't be used anymore from Grafana 10.0 onwards, so we can clean up the related code.

**Who is this feature for?**

Devs

**Which issue(s) does this PR fix?**:

Related to https://github.com/grafana/grafana/issues/68076

**Special notes for your reviewer:**

Tests that I have manually done:
* check that team list and individual team pages work as expected for Enterprise/OSS Grafana;
* check that team role picker works as expected for enterprise Grafana;
* check that users can create teams when `editorsCanAdmin` is set with Enterprise/OSS Grafana.

Note that I have removed a bunch of legacy access control tests, but I don't think we have alternatives with them for RBAC. Specifically, for team member listing. I could use some help with writing those.
